dev-support/Jenkinsfile

```diff
@@ -71,7 +71,7 @@ pipeline {
                     checkout([
                         $class: 'GitSCM',
                         branches: [[name: "${env.YETUS_VERSION}"]],
-                        userRemoteConfigs: [[ url: 'https://github.com/apache/yetus.git']]]
+                        userRemoteConfigs: [[ url: 'https://github.com/GauthamBanasandra/yetus.git']]]
```

Comment:
Debugging

Reply:
Yes, I'm trying to explore how I can get the hadolint `info` messages to be ignored - https://github.com/GauthamBanasandra/yetus/commit/e1096ecbbcbb0697473a3e34358ee28ca091a3fd.